tokio/src/sync/mpsc/bounded.rs

```diff
@@ -150,6 +152,43 @@ impl<T> Receiver<T> {
         poll_fn(|cx| self.poll_recv(cx)).await
     }
 
+    /// Blocking receive to call outside of asynchronous contexts
+    ///
+    /// ```
+    /// use std::thread;
+    /// use tokio::runtime::Runtime;
+    /// use tokio::sync::mpsc;
+    ///
+    /// fn main() {
+    ///     let (mut tx, mut rx) = mpsc::channel::<u8>(10);
+    ///     
+    ///     let sync_code = thread::spawn(move || {
+    ///         assert_eq!(Ok(Some(10)), rx.blocking_recv());
+    ///     });
+    ///
+    ///     Runtime::new()
+    ///         .unwrap()
+    ///         .block_on(async move {
+    ///             let _ = tx.send(10).await;
+    ///         });
+    ///     sync_code.join().unwrap()
+    /// }
+    /// ```
+    #[cfg(feature = "blocking")]
```

Comment:
I don't think you need a feature flag cfg here. The `blocking` cfg flag is just for enabling the runtime's blocking pool.